src/components/Home/0A-HomePage-Render.jsx

```diff
@@ -0,0 +1,24 @@
+import { Trending } from './1B-Trending'
+import { AllArticles } from './2B-All-Articles'
+import { useState } from 'react'
+import { useEffect } from 'react'
+import { getAllArticles } from '../../utils'
+
+export const HomePageRender = () =>
+{
+    const [CurrentArticles, SetCurrentArticles] = useState([])
+    useEffect(() => 
+    {
+        getAllArticles()
+        .then(({ data }) =>
+        {
+            SetCurrentArticles([...data])
+        })
+    }, [])
+    return(
+        <div id="Home_Page_Render">
+            <Trending CurrentArticles={CurrentArticles}/>
```

Comment:
Really great idea, but keep the functionality to the spec on the ticket! This is not a requirement at the moment.